radicle/src/git.rs

```diff
@@ -106,6 +113,48 @@ pub mod refs {
         pub fn id(remote: &RemoteId) -> Namespaced {
             IDENTITY_BRANCH.with_namespace(remote.into())
         }
+
+        /// The collaborative object reference, identified by `typename` and `object_id`, under the given `remote`.
+        ///
+        /// `refs/namespaces/<remote>/refs/cobs/<typename>/<object_id>`
+        ///
+        pub fn cob<'a>(
+            remote: &RemoteId,
+            typename: &cob::TypeName,
+            object_id: &cob::ObjectId,
+        ) -> Namespaced<'a> {
+            Qualified::from_components(
+                component!("cobs"),
```

Comment:
I sort of recall seeing a ref somewhere that had a singular `cob` in it. Just want to flag that in case there's a mismatch somewhere.

Reply:
cool, yes agreed :+1: